### PR TITLE
[Android] Don't assume casts to Activity from Context will be successful

### LINKED ISSUE #1407: [Previewer] `SetDefaultActionBarTitleTextColor` prevents the Android previewer from working
### Description

The `SetDefaultActionBarTitleTextColor` method (or any method for that matter) should not make assumption about a context being an `Activity` (that is never do `((Activity)_context)`).

It should use the as operator and null-check the result instead (or `?.` operator).

### Steps to Reproduce

1. Open a ContentPage xaml file with master Forms Previewer.
2. Exception happens.

### Expected Behavior

Works

### Actual Behavior

Exception:

```
ERROR: : AndroidXamlRenderer had a rendering error: System.InvalidCastException: Specified cast is not valid.
at Xamarin.Forms.Platform.Android.Platform.SetDefaultActionBarTitleTextColor () in D:\agent\_work\1\s\Xamarin.Forms.Platform.Android\Platform.cs:924
at Xamarin.Forms.Platform.Android.Platform..ctor (Android.Content.Context context, System.Boolean embedded) in D:\agent\_work\1\s\Xamarin.Forms.Platform.Android\Platform.cs:64
at Xamarin.Forms.Platform.Android.Platform..ctor (Android.Content.Context context) in D:\agent\_work\1\s\Xamarin.Forms.Platform.Android\Platform.cs:1087
```

### Screenshots

N/A

### Reproduction Link

N/A

### PR DESCRIPTION
**Pull request by hartez**
### Description of Change ###

Null check casts to Activity from Context in Platform

### Bugs Fixed ###

Theoretically fixes #1407

### CODE REVIEW

File: Xamarin.Forms.Platform.Android/Platform.cs [hunk 1]
```diff
@@ -17,6 +17,7 @@
 using Xamarin.Forms.Platform.Android.AppCompat;
 using FragmentManager = Android.Support.V4.App.FragmentManager;
 using Xamarin.Forms.Internals;
+using Debug = System.Diagnostics.Debug;
```
Comment (StephaneDelcroix) [inline Xamarin.Forms.Platform.Android/Platform.cs]: is this ever used ?

**Comment by samhouts** [comment]
Should this be pointed at 15-5 instead of master? It's on the vCurrent board.

**Comment by hartez** [comment]
> Should this be pointed at 15-5 instead of master? It's on the vCurrent board.
It was on the wrong board. The exception in #1407 only occurs in 15-6.

**Comment by samhouts** [comment]
um...was this supposed to be merged?

**Comment by hartez** [comment]
> um...was this supposed to be merged?
Not really, but it's not tragic. The changes here are basically superseded by #1592.

**Pull request merged**